### ISSUE TITLE
Cannot save workflow if one of credentials was changed

### ISSUE THREAD

**Issue by mecurc** [n8n-io/n8n#949]
**Describe the bug**
cannot save workflow if one of credentials was changed.

**To Reproduce**
1. Create any credential 
2. Create a workflow that use 1.
3. Rename credential
4. Open workflow from step 2, change used credential to new one and see error here.

Error message:
```
There was a problem deleting the workflow:
Could not find credentials for type "airtableApi" with name "OLD CREDENTIAL NAME HERE".
```

And when I try to save workflow with a different name, it is successfully saved.

**Expected behavior**
I can save workflow

**Environment (please complete the following information):**
 - n8n Version [e.g. 0.81.0]

**Comment by janober** [comment]
I tried now a few times. For me, it always works without a problem. Can you please be more specific what exactly you are doing? Like do you close the open workflow? Do you at any point refresh the window? How do you create the credentials, via the node or via the menu? Where do you rename the credentials, via the node or the menu? ....?

**Comment by the-mcnaveen** [comment]
Here is how I'm faced the same issue.

The Workflow editor is opened in Chrome Profile 1.
I went to another chrome profile (2) and opened the workflow editor just to Create and Authenticate Google API
(After your reply in forum: https://community.n8n.io/t/if-json-response-is-null-send-it-to-true-branch/2315/7?u=mcnaveen)

Because that specific email address is not logged in the chrome profile 1.

When finished adding the Credentials, **I refreshed the Chrome Profile 1 to reload the available credentials**. After that, I can't save the Workflow even make it Inactive, and can't delete it too.

This is the only error I'm getting.

![Selection_203](https://user-images.githubusercontent.com/8493007/93036064-629b7200-f65c-11ea-9414-1082713b8032.png)


So, I tried to save the workflow with different names and it worked.

**Comment by mecurc** [comment]
I encounter this problem when I change credentials not directly in workflow but through the menu on the left.

1. Open credentials in left menu
2. Open used credential and change something(or just rename)
3. Open workflow and update credential (to new name)
4. Try save. Error here.

**Comment by the-mcnaveen** [comment]
Here is the fix for that. Which personally worked for me now.

I created a New Google API Credential because if you look closely at the image I attached. It says type "googleAPI". with the Name showing there.

In my case,

- Credential Type: Google API
- Credential Name: Words Google Sheets API
- Fill random text in Email and Private key.

Save it and Refresh the page. It fixed the error for me. Not getting this anymore.
